packages/amplify-codegen-e2e-tests/test-apps/swift/swift/swiftApp.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  swiftApp.swift
+//  swift
+//
+//  Created by Pilcher, Dane on 11/21/22.
+//
+
+import SwiftUI
+import Amplify
+import AWSDataStorePlugin
+
+@main
+struct swiftApp: App {
+    init() {
+        configureAmplify()
+    }
+    var body: some Scene {
+        WindowGroup {
+            ContentView()
+        }
+    }
+}
+
+func configureAmplify() {
+    let dataStorePlugin = AWSDataStorePlugin(modelRegistration: AmplifyModels())
+    do {
+        try Amplify.add(plugin: dataStorePlugin)
+        try Amplify.configure()
+        print("Initialized Amplify");
+    } catch {
+        // simplified error handling for the tutorial
+        print("Could not initialize Amplify: \(error)")
+    }
```

Comment:
Should this just throw for the purposes of our test app?

Reply:
Actually, I guess runtime errors wouldn't be caught in the current setup, right?

Reply:
Yeah, the test does not assert against runtime, only build time.